admin_workspace_integration_test.go

```diff
@@ -14,6 +14,76 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestAdminWorkspaces_ListWithFilter(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	org, orgCleanup := createOrganization(t, client)
+	defer orgCleanup()
+
+	wTest1, wTest1Cleanup := createWorkspace(t, client, org)
+	defer wTest1Cleanup()
+
+	wTest2, wTest2Cleanup := createWorkspace(t, client, org)
+	defer wTest2Cleanup()
+
+	t.Run("when filtering workspaces on a current run status", func(t *testing.T) {
+		_, appliedCleanup := createRunApply(t, client, wTest1)
+		t.Cleanup(appliedCleanup)
+
+		_, unAppliedCleanup := createRunUnapplied(t, client, wTest2)
+		t.Cleanup(unAppliedCleanup)
+
+		wl, err := client.Admin.Workspaces.List(ctx, &AdminWorkspaceListOptions{
+			Filter: string(RunApplied), Include: []AdminWorkspaceIncludeOpt{AdminWorkspaceCurrentRun},
+		})
+
+		require.NoError(t, err)
+		require.NotEmpty(t, wl.Items)
+		assert.Equal(t, wl.Items[0].CurrentRun.Status, RunApplied)
```

Comment:
Oh, now that I think this through an extra step, this is fine and doesn't need to be changed to an `assert.Contains` with our run. I got all turned around! But because the filter is limiting runs to the RunApplied status, the first run definitely has to be in status RunApplied, even if it's not the run we created. 👍🏼

Reply:
Ok!